tests/testthat/test-tmb.R

```diff
@@ -18,6 +18,38 @@ test_that("h_mmrm_tmb_control works as expected", {
   expect_identical(result, expected)
 })
 
+# h_mmrm_tmb_extract_vars ----
+
+test_that("h_mmrm_tmb_extract_vars works as expected", {
+  cl <- call("cs", quote(a | b / c ))
```

Comment:
please group into several `test_that()` with specific labels to make clearer what is tested